runtime/rasdump/javadump.cpp

```diff
@@ -3968,7 +3968,7 @@ JavaCoreDumpWriter::findThreadCycle(J9VMThread* vmThread, J9HashTable* deadlocks
 	J9ThreadAbstractMonitor *lock;
 	J9VMThread *owner;
 	UDATA status;
-	j9object_t lockObject;
+	j9object_t lockObject = NULL;
```

Comment:
While you're here, please fix the preceding three locals.

Reply:
Done!